src/app/containers/NewSignupContainer/VerificationStep/useVerification.js

```diff
@@ -0,0 +1,20 @@
+import { useConfig, useApiResult } from 'react-components';
+import { queryCheckVerificationCode, queryVerificationCode } from 'proton-shared/lib/api/user';
+
+const useVerification = () => {
+    const { CLIENT_TYPE } = useConfig();
+    const { request: requestCode } = useApiResult(({ Type, Destination }) => queryVerificationCode(Type, Destination));
+    const { request: requestVerification } = useApiResult(({ Token, TokenType }) =>
```

Comment:
If `result` or `loading` is not needed, it's better to use `useApi` to avoid the unnecessary state changes.